.github/actionlint.yaml

```diff
@@ -14,3 +14,5 @@ paths:
       #    - issue: https://github.com/rhysd/actionlint/issues/511
       #    - PR: https://github.com/rhysd/actionlint/pull/513
       - '"env" is not allowed in "runs" section because .* is a Composite action.*'
+      # Temporary workaround to fix actionlint not recognizing jobs.<job_id>.result
+      - 'property "result" is not defined in object type \{outputs: \{.*\}\}'
```

Comment:
changes look good to me except this, can we update the code so we do not have to add this?

Reply:
I am not too happy to add more exceptions here, but I think its OK got now

Reply:
@mkzie2 Could you raise the upstream issue and PR and add it here same as the comment above?